functions/FunctionsExample/FunctionsExample.xcodeproj/project.pbxproj

```diff
@@ -738,57 +720,27 @@
 			repositoryURL = "https://github.com/firebase/firebase-ios-sdk";
 			requirement = {
 				kind = upToNextMajorVersion;
-				minimumVersion = 9.0.0;
+				minimumVersion = 10.0.0;
```

Comment:
Should be 10.17.0

Reply:
In practice, the `upToNextMajorVersion` ensures that the dependency resolves to 10.17.0.

Reply:
But it allows an earlier version for an existing project, which would fail.

Reply:
Sorry, I'm confused. The `FirebaseFunctionsSwift` went away during the 9 -> 10 transition. Not the 10.16 -> 10.17 release.

Reply:
I'm confused. I started by thinking this PR was an update for the 10.17.0 consolidation, but it's not. Sorry.

Reply:
No worries, this one was just very very outdated! Same as #1527